### ISSUE TITLE
Error at "building wheel" when installing

### ISSUE THREAD

**Issue by thoraxmax** [hbldh/pymetawear#56]
* pymetawear version: most recent
* Python version: 3.7
* Operating System: Windows 10

### Description

I tried to install the build with pip in cmd run as admin, I got a build error when "building wheel". Something with the MetaWear.Win32.dll file. 

### What I Did
I have installed VS2017 community, the win to sdk version, warble and pywarble without errors. Now suddenly can't get any further. than the errors from the command in cmd. 

Help is very much appreciated, I am new to programming but I am studying it a lot right now. I am very exited to use python with my metamotion! 
/Greetings from Sweden
```
C:\WINDOWS\system32>py -m pip install pymetawear
Collecting pymetawear
  Using cached https://files.pythonhosted.org/packages/c9/cc/dbc0eede0db795848ffd3b51a170a818df40420466b256361a66b9281e1c/pymetawear-0.11.0-py2.py3-none-any.whl
Requirement already satisfied: tqdm in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from pymetawear) (4.25.0)
Collecting metawear==0.5.0 (from pymetawear)
  Using cached https://files.pythonhosted.org/packages/cc/47/ace0feea8cb880b4dfe4b7a2099a0b89cc9a76acdaa23ea87d25bbcdc2ff/metawear-0.5.0.tar.gz
Requirement already satisfied: warble<2.0,>=1.1 in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from metawear==0.5.0->pymetawear) (1.1.0)
Requirement already satisfied: requests in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from metawear==0.5.0->pymetawear) (2.19.1)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from requests->metawear==0.5.0->pymetawear) (2018.8.24)
Requirement already satisfied: urllib3<1.24,>=1.21.1 in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from requests->metawear==0.5.0->pymetawear) (1.23)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from requests->metawear==0.5.0->pymetawear) (3.0.4)
Requirement already satisfied: idna<2.8,>=2.5 in c:\users\thors\appdata\local\programs\python\python37-32\lib\site-packages (from requests->metawear==0.5.0->pymetawear) (2.7)
Building wheels for collected packages: metawear
  Running setup.py bdist_wheel for metawear ... error
  Complete output from command C:\Users\thors\AppData\Local\Programs\Python\Python37-32\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\thors\\AppData\\Local\\Temp\\pip-install-th0eyhsa\\metawear\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d C:\Users\thors\AppData\Local\Temp\pip-wheel-ug99r3st --python-tag cp37:
  running bdist_wheel
  running build
  running build_py
  Microsoft (R) Build Engine version 15.8.168+ga8fba1ebd7 for .NET Framework
  Copyright (C) Microsoft Corporation. All rights reserved.

  Build started 2018-09-13 18:29:08.
  Project "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" on node 1 (default targets).
  PrepareForBuild:
    Creating directory "build\Win32\Release\MetaWear.Win32\".
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppBuild.targets(399,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    Creating directory "dist\Release\lib\Win32\".
    Creating directory "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\".
  InitializeBuildStatus:
    Creating "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
  ClCompile:
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\bin\HostX86\x86\CL.exe /c /Zi /nologo /W3 /WX- /diagnostics:classic /O2 /Oi /Oy- /GL /D METAWEAR_DLL_EXPORTS /D _WINDLL /D _UNICODE /D UNICODE /Gm- /EHsc /MD /GS /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"build\Win32\Release\MetaWear.Win32\\" /Fd"build\Win32\Release\MetaWear.Win32\vc141.pdb" /Gd /TP /analyze- /FC /errorReport:queue src\metawear\core\cpp\anonymous_datasignal.cpp src\metawear\core\cpp\datasignal.cpp src\metawear\core\cpp\debug.cpp src\metawear\core\cpp\event.cpp src\metawear\core\cpp\logging.cpp src\metawear\core\cpp\macro.cpp src\metawear\core\cpp\moduleinfo.cpp src\metawear\core\cpp\responseheader.cpp src\metawear\core\cpp\settings.cpp src\metawear\core\cpp\timer.cpp src\metawear\core\cpp\version.cpp src\metawear\dfu\cpp\dfu_operations.cpp src\metawear\dfu\cpp\dfu_operations_details.cpp src\metawear\dfu\cpp\dfu_utility.cpp src\metawear\dfu\cpp\file_operations.cpp src\metawear\dfu\cpp\miniz.cpp src\metawear\impl\cpp\datainterpreter.cpp src\metawear\impl\cpp\metawearboard.cpp src\metawear\peripheral\cpp\haptic.cpp src\metawear\peripheral\cpp\ibeacon.cpp src\metawear\peripheral\cpp\led.cpp src\metawear\peripheral\cpp\neopixel.cpp src\metawear\platform\cpp\async_creator.cpp src\metawear\platform\cpp\memory.cpp src\metawear\platform\cpp\task.cpp src\metawear\platform\cpp\threadpool.cpp src\metawear\processor\cpp\dataprocessor.cpp src\metawear\processor\cpp\dataprocessor_config.cpp src\metawear\sensor\cpp\accelerometer.cpp src\metawear\sensor\cpp\accelerometer_bosch.cpp src\metawear\sensor\cpp\accelerometer_mma8452q.cpp src\metawear\sensor\cpp\ambientlight_ltr329.cpp src\metawear\sensor\cpp\barometer_bosch.cpp src\metawear\sensor\cpp\colordetector_tcs34725.cpp src\metawear\sensor\cpp\conductance.cpp src\metawear\sensor\cpp\gpio.cpp src\metawear\sensor\cpp\gyro_bmi160.cpp src\metawear\sensor\cpp\humidity_bme280.cpp src\metawear\sensor\cpp\magnetometer_bmm150.cpp src\metawear\sensor\cpp\multichanneltemperature.cpp src\metawear\sensor\cpp\proximity_tsl2671.cpp src\metawear\sensor\cpp\sensor_fusion.cpp src\metawear\sensor\cpp\serialpassthrough.cpp src\metawear\sensor\cpp\switch.cpp src\metawear\sensor\cpp\utils.cpp
    anonymous_datasignal.cpp
    datasignal.cpp
    debug.cpp
    event.cpp
    logging.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\logging.cpp(318): warning C4244: 'initializing': conversion from 'float' to 'uint32_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\logging.cpp(674): warning C4244: '=': conversion from 'double' to 'float', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    macro.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\macro.cpp(41): warning C4267: 'argument': conversion from 'size_t' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    moduleinfo.cpp
    responseheader.cpp
    settings.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\settings.cpp(89): warning C4244: '/=': conversion from 'float' to 'uint16_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    timer.cpp
    version.cpp
    dfu_operations.cpp
    dfu_operations_details.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\dfu_operations_details.cpp(76): warning C4244: 'initializing': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    dfu_utility.cpp
    file_operations.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(43): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(76): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(157): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    miniz.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\miniz.cpp(4867): warning C4244: '=': conversion from 'mz_uint64' to 'size_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    datainterpreter.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\impl\cpp\datainterpreter.cpp(265): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\program files (x86)\windows kits\10\include\10.0.17134.0\ucrt\stdio.h(1774): note: see declaration of 'sprintf'
    metawearboard.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\impl\cpp\metawearboard.cpp(651): warning C4267: '=': conversion from 'size_t' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    haptic.cpp
    ibeacon.cpp
    Compiling...
    led.cpp
    neopixel.cpp
    async_creator.cpp
    memory.cpp
    task.cpp
    threadpool.cpp
    dataprocessor.cpp
    dataprocessor_config.cpp
    accelerometer.cpp
    accelerometer_bosch.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\accelerometer_bosch.cpp(529): warning C4244: '=': conversion from 'float' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\accelerometer_bosch.cpp(534): warning C4244: '=': conversion from 'float' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    accelerometer_mma8452q.cpp
    ambientlight_ltr329.cpp
    barometer_bosch.cpp
    colordetector_tcs34725.cpp
    conductance.cpp
    gpio.cpp
    gyro_bmi160.cpp
    humidity_bme280.cpp
    magnetometer_bmm150.cpp
    multichanneltemperature.cpp
    Compiling...
    proximity_tsl2671.cpp
    sensor_fusion.cpp
    serialpassthrough.cpp
    switch.cpp
    utils.cpp
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\utils.cpp(20): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\program files (x86)\windows kits\10\include\10.0.17134.0\ucrt\stdio.h(1774): note: see declaration of 'sprintf'
  c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\utils.cpp(26): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\program files (x86)\windows kits\10\include\10.0.17134.0\ucrt\string.h(133): note: see declaration of 'strcpy'
  Link:
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\bin\HostX86\x86\link.exe /ERRORREPORT:QUEUE /OUT:"dist\Release\lib\Win32\MetaWear.Win32.dll" /INCREMENTAL:NO /NOLOGO kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib uuid.lib odbc32.lib odbccp32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /DEBUG /PDB:"dist\Release\lib\Win32\MetaWear.Win32.pdb" /SUBSYSTEM:WINDOWS /OPT:REF /OPT:ICF /LTCG:incremental /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"dist\Release\lib\Win32\MetaWear.Win32.lib" /MACHINE:X86 /SAFESEH /DLL build\Win32\Release\MetaWear.Win32\anonymous_datasignal.obj
    build\Win32\Release\MetaWear.Win32\datasignal.obj
    build\Win32\Release\MetaWear.Win32\debug.obj
    build\Win32\Release\MetaWear.Win32\event.obj
    build\Win32\Release\MetaWear.Win32\logging.obj
    build\Win32\Release\MetaWear.Win32\macro.obj
    build\Win32\Release\MetaWear.Win32\moduleinfo.obj
    build\Win32\Release\MetaWear.Win32\responseheader.obj
    build\Win32\Release\MetaWear.Win32\settings.obj
    build\Win32\Release\MetaWear.Win32\timer.obj
    build\Win32\Release\MetaWear.Win32\version.obj
    build\Win32\Release\MetaWear.Win32\dfu_operations.obj
    build\Win32\Release\MetaWear.Win32\dfu_operations_details.obj
    build\Win32\Release\MetaWear.Win32\dfu_utility.obj
    build\Win32\Release\MetaWear.Win32\file_operations.obj
    build\Win32\Release\MetaWear.Win32\miniz.obj
    build\Win32\Release\MetaWear.Win32\datainterpreter.obj
    build\Win32\Release\MetaWear.Win32\metawearboard.obj
    build\Win32\Release\MetaWear.Win32\haptic.obj
    build\Win32\Release\MetaWear.Win32\ibeacon.obj
    build\Win32\Release\MetaWear.Win32\led.obj
    build\Win32\Release\MetaWear.Win32\neopixel.obj
    build\Win32\Release\MetaWear.Win32\async_creator.obj
    build\Win32\Release\MetaWear.Win32\memory.obj
    build\Win32\Release\MetaWear.Win32\task.obj
    build\Win32\Release\MetaWear.Win32\threadpool.obj
    build\Win32\Release\MetaWear.Win32\dataprocessor.obj
    build\Win32\Release\MetaWear.Win32\dataprocessor_config.obj
    build\Win32\Release\MetaWear.Win32\accelerometer.obj
    build\Win32\Release\MetaWear.Win32\accelerometer_bosch.obj
    build\Win32\Release\MetaWear.Win32\accelerometer_mma8452q.obj
    build\Win32\Release\MetaWear.Win32\ambientlight_ltr329.obj
    build\Win32\Release\MetaWear.Win32\barometer_bosch.obj
    build\Win32\Release\MetaWear.Win32\colordetector_tcs34725.obj
    build\Win32\Release\MetaWear.Win32\conductance.obj
    build\Win32\Release\MetaWear.Win32\gpio.obj
    build\Win32\Release\MetaWear.Win32\gyro_bmi160.obj
    build\Win32\Release\MetaWear.Win32\humidity_bme280.obj
    build\Win32\Release\MetaWear.Win32\magnetometer_bmm150.obj
    build\Win32\Release\MetaWear.Win32\multichanneltemperature.obj
    build\Win32\Release\MetaWear.Win32\proximity_tsl2671.obj
    build\Win32\Release\MetaWear.Win32\sensor_fusion.obj
    build\Win32\Release\MetaWear.Win32\serialpassthrough.obj
    build\Win32\Release\MetaWear.Win32\switch.obj
    build\Win32\Release\MetaWear.Win32\utils.obj
       Creating library dist\Release\lib\Win32\MetaWear.Win32.lib and object dist\Release\lib\Win32\MetaWear.Win32.exp
    Generating code
    All 9417 functions were compiled because no usable IPDB/IOBJ from previous compilation was found.
    Finished generating code
    MetaWear.Win32.vcxproj -> C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\dist\Release\lib\Win32\MetaWear.Win32.dll
  FinalizeBuildStatus:
    Deleting file "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\unsuccessfulbuild".
    Touching "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\MetaWear.Win32.lastbuildstate".
  Done Building Project "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" (default targets).

  Build succeeded.

  "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" (default target) (1) ->
  (PrepareForBuild target) ->
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppBuild.targets(399,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]


  "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" (default target) (1) ->
  (ClCompile target) ->
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\logging.cpp(318): warning C4244: 'initializing': conversion from 'float' to 'uint32_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\logging.cpp(674): warning C4244: '=': conversion from 'double' to 'float', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\macro.cpp(41): warning C4267: 'argument': conversion from 'size_t' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\core\cpp\settings.cpp(89): warning C4244: '/=': conversion from 'float' to 'uint16_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\dfu_operations_details.cpp(76): warning C4244: 'initializing': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(43): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(76): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\file_operations.cpp(157): warning C4244: '=': conversion from 'double' to 'int', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\dfu\cpp\miniz.cpp(4867): warning C4244: '=': conversion from 'mz_uint64' to 'size_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\impl\cpp\datainterpreter.cpp(265): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\impl\cpp\metawearboard.cpp(651): warning C4267: '=': conversion from 'size_t' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\accelerometer_bosch.cpp(529): warning C4244: '=': conversion from 'float' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\accelerometer_bosch.cpp(534): warning C4244: '=': conversion from 'float' to 'uint8_t', possible loss of data [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\utils.cpp(20): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    c:\users\thors\appdata\local\temp\pip-install-th0eyhsa\metawear\metawear-sdk-cpp\src\metawear\sensor\cpp\utils.cpp(26): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]

      16 Warning(s)
      0 Error(s)

  Time Elapsed 00:00:40.82
  error: [Errno 2] No such file or directory: 'C:\\Users\\thors\\AppData\\Local\\Temp\\pip-install-th0eyhsa\\metawear\\MetaWear-SDK-Cpp\\dist\\release\\lib\\x86\\MetaWear.Win32.dll'

  ----------------------------------------
  Failed building wheel for metawear
  Running setup.py clean for metawear
Failed to build metawear
Installing collected packages: metawear, pymetawear
  Running setup.py install for metawear ... error
    Complete output from command C:\Users\thors\AppData\Local\Programs\Python\Python37-32\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\thors\\AppData\\Local\\Temp\\pip-install-th0eyhsa\\metawear\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\thors\AppData\Local\Temp\pip-record-cilr31n9\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    Microsoft (R) Build Engine version 15.8.168+ga8fba1ebd7 for .NET Framework
    Copyright (C) Microsoft Corporation. All rights reserved.

    Build started 2018-09-13 18:29:50.
    Project "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" on node 1 (default targets).
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppBuild.targets(399,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]
    InitializeBuildStatus:
      Creating "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
    ClCompile:
      All outputs are up-to-date.
    Link:
      All outputs are up-to-date.
      MetaWear.Win32.vcxproj -> C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\dist\Release\lib\Win32\MetaWear.Win32.dll
    FinalizeBuildStatus:
      Deleting file "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\unsuccessfulbuild".
      Touching "build\Win32\Release\MetaWear.Win32\MetaWear.Win32.tlog\MetaWear.Win32.lastbuildstate".
    Done Building Project "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" (default targets).

    Build succeeded.

    "C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj" (default target) (1) ->
    (PrepareForBuild target) ->
      C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppBuild.targets(399,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\MetaWear-SDK-Cpp\MetaWear.Win32.vcxproj]

        1 Warning(s)
        0 Error(s)

    Time Elapsed 00:00:00.40
    error: [Errno 2] No such file or directory: 'C:\\Users\\thors\\AppData\\Local\\Temp\\pip-install-th0eyhsa\\metawear\\MetaWear-SDK-Cpp\\dist\\release\\lib\\x86\\MetaWear.Win32.dll'

    ----------------------------------------
Command "C:\Users\thors\AppData\Local\Programs\Python\Python37-32\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\thors\\AppData\\Local\\Temp\\pip-install-th0eyhsa\\metawear\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\thors\AppData\Local\Temp\pip-record-cilr31n9\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\thors\AppData\Local\Temp\pip-install-th0eyhsa\metawear\

C:\WINDOWS\system32>
```

**Comment by qutang** [comment]
the metawear package version should be bumped to 0.7.0 to solve this problem. Unfortunately the author seems not maintaining the repository any more.

**Comment by hbldh** [comment]
This is solved in the v0.12.0 release.
